src/Command/CreateAdminCommand.php

```diff
@@ -11,14 +11,14 @@
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
-use Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface;
+use Symfony\Component\PasswordHasher\Hasher\UserPasswordHasherInterface;
```

Comment:
Quand on démarre le docker on a cette erreur qui revient en boucle : 

2024-04-24 00:17:20 The database is not up or not reachable:
2024-04-24 00:17:20 
2024-04-24 00:17:20 In DefinitionErrorExceptionPass.php line 48:
2024-04-24 00:17:20                                                                                
2024-04-24 00:17:20   Cannot autowire service "App\Command\CreateAdminCommand": argument "$passwo  
2024-04-24 00:17:20   rdEncoder" of method "__construct()" has type "Symfony\Component\Security\C  
2024-04-24 00:17:20   ore\Encoder\UserPasswordEncoderInterface" but this class was not found.      
2024-04-24 00:17:20